meetings/2024-12-11.md

```diff
@@ -0,0 +1,134 @@
+* RG: I would wait a week and publish it to the community forum and on matrix.
+ACTION: next week to vote on the clarification on the charter.
+* eP: Propose making a pull request on the charter and once the conversations is settled put an item on the agenda. 
+
+ACTION: MdJ to open the pull request and we'll vote on it next week. eP volunteers to help.
+
+### Special Topic Meetings updates
+
+* eP: Last week we discussed removing unused slot from the calendar but have a permanent agenda topic to encourage organizing STMs.
+* eP: we discussed removing the calendar slot. We'd like the conversation to continue though
+  * ... I'd like to ask anyone if they'd like to plan and organize an STM. DIDs are an example, we had conversations with Niko and HZ. We also talked about deltas and CRDT.
+  * ... There's also CRDT-RDF community group. I'd be interested to have a conversation about DIDs in Jan.
+* HZ: I'll help with organizing
+* eP: let's continue the planning on matrix.
+* MdJ: I can remove it from the calendar.
+
+### Choosing a different term than C2C
+URL: https://matrix.to/#/!QxZtVBYQfMeMTnespj:gitter.im/$GTTvQgauz8PY6ky2bRSMwyFVxPceA47C52asMPYtDlY?via=gitter.im&via=matrix.org&via=chat.semantic.works
+
+- MdJ: the discussion about C2C: is it Data Formats or something else?
+  - ... it is also how you link things together
+- eP: also discovery (what about path, or just "follow your nose")
+- eP: also authorization is involved. if app not authorized, should not see any data. Even if we try to decouple, there is still dependency to auth/permissions.
+  - ... the main idea is how apps can operate on common data.
+  - ... meaningful interactions, which goes beyond just data formats
+- MdJ: client client specs? I like the name for now.
+  - ... there is no direct communication between clients. there is no realtime communicatino. no P2P. Do we have a better name?
+- eP: "app interp" suggested as name
+  - ... freedom of choice with diversity of apps, still we guarantee interop of data.
+- MdJ: we can also see the matter as : apps talking to other apps.
+- eP: hearing "client to client" could be confused with P2P. it also confusing as client is used in "client-server" but here we not necessarily talk about a client as opposed to server
+- MdJ: the apps are abiding by protocols or specs. users don't , they just "click away". so more in favor of using the "client" term. but client can be confusing, I agree. can sound to much like WebRTC. But client-server-client is also fine.
+- eP: is it a star topoly of many clients to one server, or many clients to many servers. There are more clients and more servers in the topology. It could be client2server2client2server2server2client.... etc
+- MdJ: the main question: how the data moves
+  - ... who is sending data, who is receiving data
+  - ... it doesn't matter the exact route the data takes
+  - ... let's decide on "ClientClient" spec?
+- eP: it can be confusing, very similar C2C or CC specs.
```

Comment:
```suggestion
- eP: Is it a star topology of many clients to one server, or many clients to many servers? There are more clients and more servers in the topology. It could be client2server2client2server2server2client..., etc.
- MdJ: The main question: how the data moves.
  - ... Who is sending data, who is receiving data?
  - ... It doesn't matter the exact route the data takes.
  - ... Let's decide on "ClientClient" spec?
- eP: It can be confusing, very similar C2C or CC specs.
```